js/menu.js

```diff
@@ -10,6 +10,27 @@ function init() {
   var deviceWidth = $('body').width();
   var tabletBreakPoint = 640;
 
+  Fliplet.Hooks.on('addExitAppMenuLink', function () {
+    var $exitButton = $([
+      '<li class="linked with-icon" data-fl-exit-app>',
+        '<div class="fl-menu-icon">',
+          '<i class="fa fa-sign-out"></i>',
```

Comment:
```suggestion
          '<i class="fa fa-fw fa-sign-out"></i>',
```

Let's add the `fa-fw` class.

The class should also be added to the `build.html` templates. This stands for "fixed-width", which ensures that when icons are laid out on top of each other, they are lined up nicely.

This needs to be done for all menus that support icons.

See https://fontawesome.com/v4.7.0/examples/#fixed-width

![image](https://user-images.githubusercontent.com/290733/77464184-b4c2af80-6dfe-11ea-8201-560eab599a72.png)